pages/docs/styled-system/theming/theme-tools.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Theme Tools
+description: Tools to help create your dynamic custom theme
+---
+
+The Theme Tools from ChakraUI are a collection of utilities that are for public
+use to simplify your custom theme file or styling within components using
+methods to make your design more dynamic.
+
+<ComponentLinks
+  github={{ package: 'theme-tools' }}
+  npm={{ package: '@chakra-ui/theme-tools' }}
+/>
+
+With any of these tools, you will need to install the `theme-tools` package
+separately as they are not a part of `@chakra-ui/react`.

```

Comment:
I don't think the `theme-tools` package needs to be separately installed. I tried it quickly in CodeSandbox and I was able to pull in methods from that package without having to manually install it. Haven't tried to do that in a local VSC environment yet. Is that right, @segunadebayo?